### ISSUE TITLE
Suggestion: KiCAD5 badge

### ISSUE THREAD

**Issue by hattesen** [devbisme/kicad-3rd-party-tools#72]
I propose adding a KiCAD5 badge (with a different color to the KiCAD6 badge), and add it to the tools that are known to work with KiCAD5 but NOT with KiCAD6.
That way, users of this list of tools do not have to navigate to the tool repository/site, to try to figure out if it works with KiCAD6 or not.
I would be happy to go through the list and determine if each tool should be badged KiCAD5, after which the only "badge-less" tools will be the ones where it is difficult to determine which version of KiCAD the tool supports.

One thing to consider: should tools that work with both KiCAD6 *and* KiCAD5 (possibly an earlier tool release) be badged with both versions, or should the KiCAD5 badge only be used as a warning to KiCAD6 users not to bother looking at the tool?

**Comment by Bouni** [comment]
I would suggest to tag a plugin with all the badges that it is compatible with, so in case of v5 and v6 both tese badges and if even compatible with v7, that one as well.

**Comment by hattesen** [comment]
@Bouni 
I agree that being proactive about V7 is a good idea - compatibility with v6.99 would count as v7, right?

**Comment by hattesen** [comment]
Thanks @Bouni, I agree with your views. Once @devbisme decides whether to add badges for _any_ compatible KiCad version (V5, V6, V7), I'll go ahead and add them in a PR.

Couple of clarifying questions:

1. Should the badges be added immediately after the bullet for the extension/tool, in descending order?
2. Should V5 badge be added without comment, if a prior release of the tool is compatible (rather than the current version)?
3. Are these badges/colors acceptable?
    * ![](https://img.shields.io/badge/V7-KiCad-green) `![](https://img.shields.io/badge/V7-KiCad-green)`
    * ![](https://img.shields.io/badge/V6-KiCad-blue) `![](https://img.shields.io/badge/V6-KiCad-blue)`
    * ![](https://img.shields.io/badge/V5-KiCad-orange) `![](https://img.shields.io/badge/V5-KiCad-orange)`
    * [Color](https://shields.io/#colors) may also be specified as [HEX value (without # prefix](https://css-tricks.com/snippets/css/named-colors-and-hex-equivalents/)), e.g. `![](https://img.shields.io/badge/V8-KiCad-9400D3)` ![](https://img.shields.io/badge/V8-KiCad-9400D3).
4. If it is not obvious (within a reasonable time) to determine which KiCad version a given tool supports, I could add a GitHub/GitLab issue in the tool repository asking for clarification of supported KiCad versions with a link to this repository. Is that OK?

### Sample, which actually suppports V7, V6 and V5 (possibly using old release)

- ![](https://img.shields.io/badge/V7-KiCad-green) ![](https://img.shields.io/badge/V6-KiCad-blue) ![](https://img.shields.io/badge/V5-KiCad-orange) [Interactive Html Bom](https://github.com/openscopeproject/InteractiveHtmlBom) ([demo](https://openscopeproject.org/InteractiveHtmlBomDemo/OSPx201/ibom.html)) tool designed to assist with hand assembling pcbs. Output is viewable in any modern browser and allows user to easily highlight a specific reference or all components in a group on a visual rendering of pcb. Script works both as Pcbnew action plugin and as a command line tool.

**Comment by Bouni** [comment]
@hattesen 

I totally like your proposal!

The only thing that could be improved a bit is the layout, here's what I think is much more readable:

>  -  **[Interactive Html Bom](https://github.com/openscopeproject/InteractiveHtmlBom)**
    ![](https://img.shields.io/badge/V7-KiCad-green) ![](https://img.shields.io/badge/V6-KiCad-blue) ![](https://img.shields.io/badge/V5-KiCad-orange)
  ([demo](https://openscopeproject.org/InteractiveHtmlBomDemo/OSPx201/ibom.html))
  tool designed to assist with hand assembling pcbs. Output is viewable in any modern browser and allows user to easily highlight a specific reference or all components in a group on a visual rendering of pcb. Script works both as Pcbnew action plugin and as a command line tool.
